<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewersOptionsWidget</class>
 <widget class="QWidget" name="ViewersOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Viewers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="gpLogViewer">
     <property name="title">
      <string>Log viewer</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_161">
          <property name="text">
           <string>For TXT log files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbTxtLogFiles">
          <item>
           <property name="text">
            <string>Built-in viewer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Associated program</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>External viewer</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_162">
          <property name="text">
           <string>For HTML log files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbHtmlLogFiles">
          <item>
           <property name="text">
            <string>Built-in viewer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Associated program</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>External viewer</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_165">
          <property name="text">
           <string>For  DBF log files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbDbfLogFile">
          <item>
           <property name="text">
            <string>Built-in viewer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Associated program</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>External viewer</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="gpExternalViewers">
     <property name="title">
      <string>External viewers</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblViewersTxtLogFiles">
        <property name="text">
         <string>For TXT log files</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editViewersTxtLogFiles"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnViewersTxtLogFiles">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../qhostmonitor/qHostMonitor.qrc">
            <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblViewersHtmlLogFiles">
        <property name="text">
         <string>For HTML logs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editViewersHtmlLogFiles"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnViewersHtmlLogFiles">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../qhostmonitor/qHostMonitor.qrc">
            <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblViewersDbfLogFiles">
        <property name="text">
         <string>For DBF log files</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editViewersDbfLogFiles"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnViewersDbfLogFiles">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../qhostmonitor/qHostMonitor.qrc">
            <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../qhostmonitor/qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
